code/game/objects/items/stacks/seaweed.dm

```diff
@@ -0,0 +1,19 @@
+/obj/item/stack/seaweed
+	name = "seaweed sheet"
+	desc = "Weed.. from the Sea!"
+	singular_name = "seaweed sheet"
+	icon = 'icons/obj/stacks/seaweed.dmi'
```

Comment:
Think it would fit ```stacks/organic.dmi``` instead? Not worth created a new one for just a single sprite.